src/controllers/ContinuedIllnessController.ts

```diff
@@ -0,0 +1,77 @@
+import Joi from '@hapi/joi';
+import { SessionMiddleware } from 'ch-node-session-handler';
+import { controller } from 'inversify-express-utils';
+import { BaseController } from './BaseController';
+import { FormValidator } from './validators/FormValidator';
+
+import { AuthMiddleware } from 'app/middleware/AuthMiddleware';
+import { IllnessReasonFeatureMiddleware } from 'app/middleware/IllnessReasonFeatureMiddleware';
+import { loggerInstance } from 'app/middleware/Logger';
+import { Appeal } from 'app/models/Appeal';
+import { Illness } from 'app/models/Illness';
+import { YesNo } from 'app/models/fields/YesNo';
+import { createSchema } from 'app/models/fields/YesNo.schema';
+import { CONTINUED_ILLNESS_PAGE_URI } from 'app/utils/Paths';
+import { Navigation } from 'app/utils/navigation/navigation';
+
+const template = 'continued-illness';
+
+const navigation : Navigation = {
+    previous(): string {
+        return '';
+    },
+    next(): string {
+        return '';
+    },
+    actions: (_: boolean) => {
+        return {
+            continue:'action=continue'
+        };
+    }
+};
+
+@controller(CONTINUED_ILLNESS_PAGE_URI, IllnessReasonFeatureMiddleware, SessionMiddleware, AuthMiddleware)
+export class ContinuedIllnessController extends BaseController<any> {
+
+    constructor() {
+        const errorMessage = 'You must tell us if this is a continued illness';
+        const schema: Joi.AnySchema = Joi.object({
+            continuedIllness: createSchema(errorMessage)
+        }).unknown(true);
+
+        super(
+            template,
+            navigation,
+            new FormValidator(schema)
+        );
+    }
+
+    protected prepareViewModelFromAppeal(appeal: Appeal): any {
+        const illnessStartDateRaw = appeal.reasons.illness?.illnessStart;
+
+        if (!illnessStartDateRaw) {
+            throw new Error('Illness Start Date expected but not found');
+        } else {
+            const illnessStart = illnessStartDateRaw.toLocaleDateString('en-GB', {
```

Comment:
I suppose `illnessStartDateRaw` is a raw string when it is read from session store so I do not think there is `toLocaleDateString` function available in String prototype. I expect this to fail.

Reply:
I should confirm this with Natalie. My feeling is that Date validation will be far easier if the date is in a Date format, rather than a string.

Reply:
If you guys want to use Date you need to have custom deserialisation logic cause in Redis Date will be stored as string.

Reply:
*cries*